compiler/testData/diagnostics/testsWithStdLib/contracts/controlflow/flowInlining/labeledReturns.fir.kt

```diff
@@ -42,7 +42,7 @@ fun threeLevelsReturnNoInitialization(x: Int?): Int? {
             }
         }
         // Possible to report unreachable here
-        y = 54
+        <!VAL_REASSIGNMENT!>y<!> = 54
```

Comment:
See the comments above that explain the complex structure of inner-outer entanglement. No idea yet...

Reply:
Probably FE 1.0 doesn't report reassignment here because the code is unreachable